app/Http/Controllers/License/LicenseController.php

```diff
@@ -205,17 +204,23 @@ public function updateLicensedDomain($licenseCode, $domain, $productId, $license
      */
     protected function getIpAndDomain($domain)
     {
+
         $isIP = (bool) ip2long($domain);
-        if ($isIP == true) {
-            $requiredomain = 0;
-            $ip = $domain;
-            $domain = '';
+        if ($domain != '') {
+            if ($isIP == true) {
+                $requiredomain = 0;
+                $ip = $domain;
+                $domain = '';
+            } else {
+                $requiredomain = 1;
+                $domain = $domain;
```

Comment:
what is the purpose of this line?

Reply:
If the url is in domain form then 'Allow Require Domain' option in License manager will be 1.